src/utils/jellyfin-apiclient/BackdropImage.ts

```diff
@@ -0,0 +1,52 @@
+import { BaseItemDto } from '@jellyfin/sdk/lib/generated-client';
+import { randomInt } from '../number';
+import { ApiClient } from 'jellyfin-apiclient';
```

Comment:
It looks like a couple of these could be type imports and it's generally better for the final bundle size to use the full path for SDK imports (although this is less relevant for a type import).

```suggestion
import { type BaseItemDto } from '@jellyfin/sdk/lib/generated-client/models/base-item-dto';
import { type ApiClient } from 'jellyfin-apiclient';

import { randomInt } from '../number';
```